docs/docs/deploy/existing-project.md

```diff
@@ -4,8 +4,17 @@ sidebar_label: Deploy an existing project
 sidebar_position: 0
 ---
 
-Follow this tutorial to deploy an existing Rill project to Rill Cloud. When you deploy a project, its dashboards become available online and you can invite other people to access it.
-
+Follow this tutorial to deploy an existing Rill project to Rill Cloud. Deploying a project makes its dashboards available online and enables you to invite others to access it.
+Flow diagram below shows the steps needed for deploying an existing project.  
+```mermaid
+graph LR;
+    A(Local artifacts);
```

Comment:
nit: use `Local code files` instead of `Local artifacts` (the way we use "artifact" internally isn't how people normally use it, so it probably doesn't make sense to users)